scan/src/utils/Param.re

```diff
@@ -0,0 +1,4 @@
+type t = {
```

Comment:
If we are having this type. We should change borsh to return this type as opposed to `(string, string)`? @evilpeach @taobun , thoughts?